engine/src/main/scala/cromwell/webservice/WorkflowSubmitBatchActor.scala

```diff
@@ -0,0 +1,47 @@
+package cromwell.webservice
+
+import akka.actor.{Actor, ActorRef, Props}
+import cromwell.engine.WorkflowSourceFiles
+import cromwell.engine.workflow.WorkflowManagerActor
+import cromwell.webservice.CromwellApiHandler.{WorkflowManagerBatchSubmitResponse, WorkflowManagerResponse}
+
+/** Submits a sequence of sources, then messages back the respective sequence of responses. */
+class WorkflowSubmitBatchActor(responseActor: ActorRef, requestHandlerActor: ActorRef,
+                               sources: Seq[WorkflowSourceFiles]) extends Actor {
+
+  private val responses = Array.fill[Option[WorkflowManagerResponse]](sources.size)(None)
+
+  override def preStart() = {
+    sources.zipWithIndex foreach {
+      case (source, index) =>
+        context.actorOf(
+          Props(new WorkflowSubmitIndexedActor(self, requestHandlerActor, source, index)),
+          "WorkflowSubmitIndexedActor_" + index)
+    }
```

Comment:
Not sure I completely follow, but happy to whiteboard out IRL. I thought about this for a while before re-implementing the previous commit that used asks, but had to get something to Alex asap.

> I think it's equivalent to messaging directly the WorkflowManagerActor from here…

The responses **MUST** be kept in the same order as the requested sources. The only thing the `WorkflowSubmitIndexedActor` adds is that it tracks the index order for the responses, that are _not_ guaranteed by akka to arrive in order.